src/main.ts

```diff
@@ -119,7 +119,7 @@ export function revealCursor() {
 
   // following cursor disposable
   if (view !== undefined) {
-    view.selectAtCursorLine(editor.getCursorBufferPosition())
+    view.selectAtCursorLine(editor.getCursors()[0])
```

Comment:
```suggestion
    view.selectAtCursorLine(editor.getLastCursor())
```
?

Generally, it's the last cursor we care about because it's the one most probably visible.

Reply:
Fixed in https://github.com/atom-community/atom-ide-outline/pull/98/commits/3e0c6d04dfdadd60d4bd1c32d7966ac694ade220